generators/text_cleaning/email_cleaner/README.md

```diff
@@ -0,0 +1 @@
+This modules removes certain aspects of the email to focus on the context. The aspects being removed are Sentences starting with "EXTERNAL MAIL", Brackets, that starts with "cid.image", everything after the Disclaimer or until the next Mail in case of response-mails and everything after the signiture in brackets or until the next Mail. 
```

Comment:
Typos

```suggestion
This module removes certain aspects of the email to focus on the context. The aspects being removed are sentences starting with "EXTERNAL MAIL", brackets, that start with "cid.image", everything after the disclaimer or until the next mail in case of response mail, and everything after the signature in brackets or until the next mail.
```